### DISCUSSION TITLE
Only for Data Backups or also for Linux Filesystem Snapshots?

### DISCUSSION THREAD

**Discussion by Tom-H-L** [borgbase/vorta#1460] [FAQ] [answered]
Can someone use Vorta/Borg only for user data backups or also for system file snapshots of the Linux root files system, or is it better to use another tool for that task, e.g. "Back in Time"?

**Comment by real-yfprojects** [comment, marked as answer]
You can use *borg* with root privileges for that, e.g when calling it with `sudo`. In fact many borg users use it to backup their complete file system. However we are sorry that *vorta* currently doesn't support being run as root user.
